src/rust/src/sapling.rs

```diff
@@ -164,7 +166,7 @@ impl BundleAssembler {
         spend_auth_sig: &[u8; 64],
     ) -> bool {
         // Deserialize the value commitment
-        let cv = match de_ct(jubjub::ExtendedPoint::from_bytes(cv)) {
+        let cv = match Option::from(ValueCommitment::from_bytes_not_small_order(cv)) {
```

Comment:
It's canonically checked during transaction deserialization now, instead of during proof verification: https://github.com/zcash/librustzcash/commit/23922ca290e300db3252e484dfdcd18c17ed75ee#diff-26bf613397d822f2041c500a56d5dd15016ac94c1955bfc2e85628b31c8e58efL47-R51